piou-piou/ribs-cron-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "PiouPiou\\RibsCronBundle\\RibsCronBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "IP_CRON_EXTERNAL": "",
+        "IP_CRON_INTERNAL": "127.0.0.1"
```

Comment:
These environment variables is never used.

Reply:
It is used in RibsCronController on line 41.
IP_CRON_EXTERNAL will contain allowed external IP of the server to call cron url

Reply:
It is good practice to use the environment variables as values to the bundle's configuration. Then use those values to register the services you need. 

I would highly recommend to avoid using environment variables directly in your code, especially not in the controller.

Reply:
Ok I will change that and put those variables in yaml config file